src/jdk.compiler/share/classes/com/sun/tools/javac/tree/DocPretty.java

```diff
@@ -192,9 +199,9 @@ public Void visitDocComment(DocCommentTree node, Void p) {
     @Override @DefinedBy(Api.COMPILER_TREE)
     public Void visitDocRoot(DocRootTree node, Void p) {
         try {
-            print("{");
```

Comment:
Done - but had to change the one with the `sep` argument to take a char instead of String. Seems not used elsewhere, but wasn't sure because of the `protected`

Reply:
While `DocPretty` is public, the package is not publicly exported from the `jdk.compiler` module, and there are no subtypes within the JDK code base, so I think we can ignore the `protected` for now.

If we wanted to be paranoid, it would be reasonable to change `print(char)` to something along the lines of
```
if (Character.isASCII(ch)) out.write(ch); else print((Character) ch);
```

Reply:
Should we (I in that case) be paranoid, though? I would leave it as-is. There is a big class comment above that anyone depending on the internal api uses this at his own risk.